spec/lib/api/v3/work_packages/work_package_representer_spec.rb

```diff
@@ -69,11 +69,11 @@
 
     describe 'work_package' do
       # specifiying as it used to be different
-      it { should have_json_path('spentTime') }
+      it { is_expected.to have_json_path('spentTime') }
 
-      it { should_not have_json_path('spentHours') }
+      it { is_expected.not_to have_json_path('spentHours') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.